sdk/python/feast/client.py

```diff
@@ -836,12 +836,39 @@ def ingest(
         Returns:
             str:
                 ingestion id for this dataset
+
+        Examples:
+            >>> from feast import Client
+            >>>
+            >>> feast_client = Client(core_url="localhost:6565")
+            >>> fs_df = pd.DataFrame(
+            >>>         {
+            >>>            "datetime": [pd.datetime.now()],
+            >>>            "driver": [1001],
+            >>>            "rating": [4.3],
+            >>>         }
+            >>>     )
+            >>> client.ingest("project1:driver", fs_df)
```

Comment:
`project:feature_set` is not a valid feature reference. It would be better to have a `project` key on the method, or even look at the project that the user has configured in their environment.